Documents/Positioning/Tutorial 03.xml

```diff
@@ -461,7 +461,7 @@ void main()
     
     elapsedTimeUniform = glGetUniformLocation(theProgram, "time");
     
-    GLuint loopDurationUnf = glGetUniformLocation(theProgram, "loopDuration");

```

Comment:
there was an error in the OpenGL spec where the return value was specifed as uint, so this is probably where it comes from